LeaderboardBackend/Models/Entities/ApplicationContext.cs

```diff
@@ -25,5 +27,15 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 		modelBuilder.Entity<Ban>()
 			.Property(ban => ban.CreatedAt)
 			.HasDefaultValueSql("now()");
+
+		modelBuilder.Entity<User>().HasData(
+			new User
+			{
+				Id = s_SeedAdminId,
+				Admin = true,
+				Email = "omega@star.com",
+				Password = "$2a$11$tNvA94WqpJ.O7S7D6lVMn.E/UxcFYztl3BkcnBj/hgE8PY/8nCRQe", // "3ntr0pyChaos"
+				Username = "Galactus"
```

Comment:
This is called only when `applicationContext.Database.EnsureCreated()` is called. So only when using the in-memory db, because for Postgres we use migrations.
The reason it is hardcoded is because the EF migration tool works by taking [a snapshot of the model](https://github.com/Dalet/leaderboard-backend/blob/1c43c1112435daa68541a02db75960fddc4805c4/LeaderboardBackend/Migrations/ApplicationContextModelSnapshot.cs#L328-L335) and generating migration code with the difference between the last snapshot and the current model. So if these values were to be dynamic, any change would be included in newly created migrations.

For the in-memory db, the seeded user is updated on startup with the environment variables after `EnsureCreated()`: https://github.com/Dalet/leaderboard-backend/blob/1c43c1112435daa68541a02db75960fddc4805c4/LeaderboardBackend/Program.cs#L225-L239

The migration code uses the environment variables directly when inserting the user: https://github.com/Dalet/leaderboard-backend/blob/1c43c1112435daa68541a02db75960fddc4805c4/LeaderboardBackend/Migrations/20230325191517_Users_SeedAdminUser.cs#L13-L16